underfs/obs/src/main/java/alluxio/underfs/obs/OBSUnderFileSystem.java

```diff
@@ -135,7 +138,8 @@ protected boolean createEmptyObject(String key) {
 
   @Override
   protected OutputStream createObject(String key) throws IOException {
-    return new OBSOutputStream(mBucketName, key, mClient);
+    return new OBSOutputStream(mBucketName, key, mClient,
+        mAlluxioConf.getList(PropertyKey.TMP_DIRS, ","));
```

Comment:
why do we need to change constructor? Can't we need the property directly in outstream?

Reply:
The code in here needs to be consistent with the main alluxio codebase. This is used in all the object output stream constructor.

I guess this may be due to configuration singleton or other configuration centralization changes.